packages/ckermit.rb

```diff
@@ -0,0 +1,19 @@
+require 'package'
+
+class Ckermit < Package
+  description 'C-Kermit is a combined serial and network communication software package.'
+  homepage 'http://www.kermitproject.org/ck90.html'
+  version '8.0.211'
+  source_url 'https://fossies.org/linux/privat/old/cku211.tar.gz'
+  source_sha256 '0503518ef8f225d06749f7923ae84521bc358daf353222c75c009cd040dc5e26'
+
+  depends_on 'ncurses'
+
+  def self.build
+    system 'make KFLAGS="-DCK_NCURSES -I/usr/local/include/ncurses" LNKFLAGS="-lresolv -lcrypt" linux'
```

Comment:
In general, we always want to abstract directories.  Please change `-I/usr/local/include/ncurses` to `-I#{CREW_PREFIX}/include/ncurses` and enclose the entire string with double quotes so the crew constant can expand.  Like this:
```
system "make KFLAGS='-DCK_NCURSES -I#{CREW_PREFIX}/include/ncurses' LNKFLAGS='-lresolv -lcrypt' linux"
```